### ISSUE TITLE
Failing to set focus to Zotero selector window on Windows

### ISSUE THREAD

**Issue by RayMacLewis** [mgmeyers/obsidian-zotero-integration#229]
Hey!
I am super happy with that plugin but one thing is quite annoying for me:
Whenever I want to run a Zotero Integration in Obsidian it will open the Zotero selector window:
![image](https://github.com/mgmeyers/obsidian-zotero-integration/assets/22128092/e8c067eb-6678-4d07-a04c-f6f5ce5e8c0a)

But it fails to set the focus / attention to that window which means that mostly this window is hidden behind something else and instead of searching for a Zotero entry directly I have to activate that window and then start searching.

Is there a way to change that?

Thanks
Ray

**Comment by Dimeruben** [comment]
Same problem here

**Comment by DeiseDaithi** [comment]
Issue still persists

**Comment by MSBack** [comment]
First the bad news: This has been reported before [here](https://github.com/mgmeyers/obsidian-zotero-integration/issues/4).

Then, the good news: There is a fix [here.](https://github.com/mgmeyers/obsidian-zotero-integration/issues/4#issuecomment-1566690780)

**Comment by RayMacLewis** [comment]
@MSBack You are awesome!
This partially helped. Partially because it correctly works for the new-style selector window shown above.
But for the classic selector window it does not work.

Do you have an idea on how to get this working for this case?

**Comment by MSBack** [comment]
> @MSBack You are awesome!

Thanks! I'm just sharing the gospel, because I was very hsppy to encounter this fix myself.

> Do you have an idea on how to get this working for this case?

No, I wasn't aware that there was a difference, since I don't use the classic selector. Is that the one where you can bulk import?

**Comment by RayMacLewis** [comment]
> No, I wasn't aware that there was a difference, since I don't use the classic selector. Is that the one where you can bulk import?

Yes, exactly! I like to work with a folder / collection system in Zotero and then bulk import folder content into Obsidian. So using the new style selector and then right-clicking and go to the classic style would be an option but is tedious.

**Comment by FeralFlora** [comment]
This can be closed.

**Comment by ZeyuSun** [comment]
This doesn't fully solve the problem. The search bar is comes up to the foreground but out of focus. There are lots of people reporting the same issue: https://github.com/mgmeyers/obsidian-zotero-integration/issues/280#issuecomment-2165422340

**Comment by huachuman** [comment]
Yes this is not fixed whatsoever. In fact it's worse. 

It only looks fixed because the menu is at least appearing above obsidian. 

But now I have to actually use my mouse and click on the text field so it gains focus, whereas before a simple alt-tab was all that was needed.

Is there any intention of actually fixing this?

@FeralFlora do you not get this issue in your own usage?

**Comment by kdqiu** [comment]
> First the bad news: This has been reported before [here](https://github.com/mgmeyers/obsidian-zotero-integration/issues/4).
> 
> Then, the good news: There is a fix [here.](https://github.com/mgmeyers/obsidian-zotero-integration/issues/4#issuecomment-1566690780)

Thanks for your solution. I met the same question. This [solution](https://github.com/mgmeyers/obsidian-zotero-integration/issues/4#issuecomment-1566690780) solved the position of bar, but can not help to set focus to zotero selector.
Pal,do you met this issue in your own usage? Do you have any other solutions? Thanks

**Comment by interconnectedMe** [comment]
Is there any update on this?  I can see a bunch of duplicate issues, and I'm also looking for a fix / workaround.
Rgds
DR
